packages/backend/src/graphql/schema.graphql

```diff
@@ -372,11 +372,15 @@ type AssetEdge {
 input ApproveIncomingPaymentInput {
   "Unique identifier of the incoming payment to be approved. Note: incoming payment must be PENDING."
   id: ID!
+  "The tenant UUID associated with the incoming payment."
+  tenantId: String!
 }
 
 input CancelIncomingPaymentInput {
   "Unique identifier of the incoming payment to be canceled. Note: incoming payment must be PENDING."
   id: ID!
+  "The tenant UUID associated with the incoming payment."
+  tenantId: String!
```

Comment:
Same as comment below, we don't need to specify tenantId in the mutations when doing operations for an existing resource so we can remove it from `ApproveIncomingPaymentInput` and `CancelIncomingPaymentInput`.